x/nft/keeper/msg_server_test.go

```diff
@@ -81,7 +81,7 @@ func (s *TestSuite) TestSend() {
 				Receiver: s.addrs[2].String(),
 			},
 			expErr: true,
-			errMsg: fmt.Sprintf("%s is not the owner of nft %s", s.addrs[1].String(), testID),
+			errMsg: fmt.Sprintf("is not the owner of nft %s", testID),
```

Comment:
Is dropping the leading owner okay for this message?

Reply:
reverted this, the removal was actually a bug which is now fixed.